Packages/com.unity.inputsystem/InputSystem/Editor/Settings/InputSettingsProvider.cs

```diff
@@ -116,6 +115,8 @@ public override void OnGUI(string searchContext)
                 EditorGUILayout.PropertyField(m_DefaultTapTime);
                 EditorGUILayout.PropertyField(m_DefaultSlowTapTime);
                 EditorGUILayout.PropertyField(m_DefaultHoldTime);
+                EditorGUILayout.PropertyField(m_TapRadius);
```

Comment:
Do we actually want to expose these settings at all? If yes, do we need to make them visible in the UI?

I'm wondering because "what constitutes a double tap" is a gesture clearly defined by the OS. A well designed app should not say "I have my own rules for what's a double tap, because the developer of this app thinks that this is the coolest double tap definition". So we should just use the OS standard here. The only valid reason I see an app developer wanting to change this is that we fucked up using the right defaults. Unfortunately, at least on iOS, this stuff is not really exposed - you should use their gesture detection APIs instead, which is not something we can easily do (or would want to). But i guess we can experiment to find the values they use (and do the same for android and windows touch), and then use those values?

Reply:
Agree on the "shouldn't be per-app kinda thing" and admit to the exposed more out of insecurity part :) Think I'm fine with removing it from the UI, though at the same time, I'm hesitant a bit given ATM it's the only discoverable way for the user to counteract a bad choice on my end.

Reply:
But our insecurity about the choices we made should not have to be burdened by users who then have to think about it. I'd rather not have these at all, and say we need to get them right (and treat it as a bug when we don't). I guess for the radius, getting it right may be tricky, though as it depends on the resolution used. So I'm ok with having the APIs for them, but let's hide them in the UI?

Reply:
Ok, can do.

Do wonder whether we are doing the right thing. LeanTouch, for example, has all these configuration values as inspector-settable properties: https://carloswilkes.github.io/Documentation/LeanTouch#LeanTouch.

Reply:
Clearly this is an area where I hold strong opinions, and not everyone has to agree :) 

But in general: lot of developer tools have a lot of settings. Like, whenever in doubt, make it a setting. I believe that this is wrong. And I believe that one of Unity's biggest strengths has always been that we make the user experience of the developer not an after-thought. So my approach is always: When in doubt that we need this, _don't_ add a setting. We can easily add a setting if it turns out we were wrong, but not the other way around.

Reply:
Yeah, disagree on that one the same way I disagree every time Apple doesn't allow me to configure something in order to "make my user experience better". And there I am finding myself typing obscure "defaults write -g NSWhatTheShitNot -float 0.03" in the shell because someone opted to "spare me the UI setting" :)

Aaaanyway, I'll go remove the UI parts.